plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveModule.java

```diff
@@ -89,6 +90,7 @@ public void configure(Binder binder)
                 .setDefault().toInstance(ImmutableList::of);
 
         binder.bind(TrinoS3ClientFactory.class).in(Scopes.SINGLETON);
+        binder.bind(S3SelectStats.class).toInstance(new S3SelectStats());
```

Comment:
What's the purpose of this? The metrics aren't exposed anywhere. Do you intend to export these with JMX?

Reply:
I did intend that, I will go over this once again and make the metric system proper. Although, do we really need to collect these metrics? If not, we can remove this and avoid the complexity entirely.